moped-database/views/project_list_view.sql

```diff
@@ -158,7 +155,15 @@ AS WITH project_person_list_lookup AS (
      LEFT JOIN moped_users added_by_user ON mp.added_by = added_by_user.user_id
      LEFT JOIN current_phase_view current_phase on mp.project_id = current_phase.project_id
      LEFT JOIN moped_public_process_statuses mpps ON mpps.id = mp.public_process_status_id
-     left join child_project_lookup cpl on cpl.parent_id = mp.project_id
+     LEFT JOIN child_project_lookup cpl on cpl.parent_id = mp.project_id
+     LEFT JOIN LATERAL
+      (
+        SELECT mpn.project_note, mpn.date_created
+        FROM moped_proj_notes mpn
+        WHERE mpn.project_id = mp.project_id AND mpn.project_note_type = 2 AND mpn.is_deleted = false
+        ORDER BY mpn.date_created DESC
```

Comment:
i wonder if we should index `moped_proj_notes.date_created`. we have an index on `moped_proj_notes.project_id` i think that's doing the heavy lifting....

Reply:
I tested before/after query time with production data and there is no noticeable impact on query speed—if anything this lateral join is a speed boost! 🙌